README.md

```diff
@@ -101,3 +101,16 @@ We use [Google Java format](https://github.com/google/google-java-format) to for
 ```bash
 ./format.sh
 ````
+
+# Snowpipe Streaming Example (Still in Preview)
+
+Run File `SnowflakeStreamingIngestExample.java` which performs following operations.
+1. Reads a JSON file which contains details regarding Snowflake Account, DB, Schema etc. Take a look at `profile_streaming.json.example` for more details.
+2. Creates a `SnowflakeStreamingIngestClient` which can be used to open one or more Streaming Channels against a table.
+3. Creates a `SnowflakeStreamingIngestChannel` against a DB, schema and table name.
+   1. Please note: A Table is expected to be present before opening a Channel. Use following SQL query and place this table name in `profile_streaming.json` file
+```sql
+create or replace table MY_TABLE(c1 number);
```

Comment:
I think we have to do other steps here as well: like create database and schema.
And also `private_key`, or  is it clearly explained somewhere else?